apisix/schema_def.lua

```diff
@@ -807,6 +807,26 @@ local xrpc_protocol_schema = {
             description = "protocol-specific configuration",
             type = "object",
         },
+        logger = {
```

Comment:
Better to use array so that we can add multiple logger in the future

Reply:
update